lib/commands/alert.js

```diff
@@ -94,16 +94,33 @@ commands.mobileHandleAlert = async function (opts = {}) {
 };
 
 helpers.getAlert = async function () {
-  let possibleAlert = await this.findNativeElementOrElements('class name', 'XCUIElementTypeScrollView', true);
+  // ScrollView should always be found, unless Safari isn't even running.
+  let possibleAlert = await this.findNativeElementOrElements(
+    'class name', 'XCUIElementTypeScrollView', true
+  );
```

Comment:
Why don't we count this one too?

Reply:
The `ScrollView` should _always_ be there, if Safari is open.  We also require the actual element, as it's the parent element used for subsequent (descendant) queries.  

So counting first would only potentially improve performance in the rare case of Safari not being open (e.g. whilst we're setting up the certificate for a new simulator).  I also imagine that extracting/constructing 0 elements is not much different than counting 0 elements, performance-wise.

So in the majority of cases, counting `ScrollView`'s first would only make performance slightly worse.